util/fipstools/delocate/delocate.go

```diff
@@ -2418,7 +2418,8 @@ func localEntryName(name string) string {
 func isSynthesized(symbol string, processor processorType) bool {
 	SymbolisSynthesized := strings.HasSuffix(symbol, "_bss_get") ||
 		symbol == "OPENSSL_ia32cap_get" ||
-		symbol == "BORINGSSL_bcm_text_hash"
+		symbol == "BORINGSSL_bcm_text_hash" ||
```

Comment:
what I don't quite get is that I removed this line `symbol == "BORINGSSL_bcm_text_hash" ||` in https://github.com/aws/aws-lc/commit/c21a05c3ea120ea0f3666fe07ee92c1dd3f73dc6#diff-3a7ed42c3edacaf0a2c43110d7da1759e6386e9ba0734df5d078475b46e47105L2421 which is already merged in, so it's not present in main (https://github.com/aws/aws-lc/blob/850af98235851dca40acae72537a4d8f97265606/util/fipstools/delocate/delocate.go#L2428). And GH doesn't report a merge conflict but rather adds the line back.